core/dynlib/lib_windows.odin

```diff
@@ -19,6 +19,7 @@ unload_library :: proc(library: Library) -> bool {
 
 symbol_address :: proc(library: Library, symbol: string) -> (ptr: rawptr, found: bool) {
 	c_str := strings.clone_to_cstring(symbol, context.temp_allocator)
```

Comment:
You'll notice here that `clone_to_cstring` is called with the explicit `context.temp_allocator` parameter, meaning there is no memory that needs freeing. Doing so would result in a bad free.